miniwallet/config/getConfig.ts

```diff
@@ -0,0 +1,25 @@
+import * as fs from 'fs'
+import * as path from 'path'
+
+export async function getConfig (networkName, configID) {
+// Get the deployment configuration
+// If a network specific config exists use that otherwise use the default
+  console.log(`Deploying to network: ${networkName}`)
+  let configPath = path.join(__dirname, './' + configID + '_' + networkName + '.ts')
+  let config
+  if (fs.existsSync(configPath)) {
+    configPath = configPath.substring(0, configPath.length - 3)
+    const { default: configNetwork } = await import(configPath)
```

Comment:
could be used to executed malicious script in temporary / mounted / remote folders